src/simulations/train_koval.py

```diff
@@ -0,0 +1,20 @@
+import pickle
+import dill as pickle
+
+from sklearn.model_selection import train_test_split
+
+from src.data.read_wfsim import f_w, W_t
+from src.helpers.features import production_rate_dataset
+from src.helpers.models import serialized_model_path
+from src.models.koval import Koval
+
+X = W_t[:-1]
+y = f_w[1:]
```

Comment:
There has to be a better way. This is not DRY, put something in the `src/helpers/features.py` file